lib/dal-test/src/helpers/change_set.rs

```diff
@@ -0,0 +1,122 @@
+//! This module provides [`ChangeSetTestHelpers`].
+
+#![warn(
+    clippy::missing_panics_doc,
+    clippy::panic,
+    clippy::panic_in_result_fn,
+    clippy::unwrap_in_result,
+    clippy::unwrap_used
+)]
```

Comment:
Someday, these should move up to the crate level. Today, we'll just make sure the `ChangeSetTestHelpers` obey these.